modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.scss

```diff
@@ -84,29 +70,43 @@
 		background-repeat: repeat-x;
 		background-size: 6px 1px;
 		bottom: 0;
-		box-sizing: border-box;
 		content: '';
 		height: 1px;
 		left: 0;
 		position: absolute;
 		right: 0;
 	}
```

Comment:
Where are you using this class? I couldn't find the file.

Reply:
I am using in  ...policies/pages/components/PolicyDetail/index.tsx